buildchow/templates/buildflow-template.yaml

```diff
@@ -0,0 +1,63 @@
+- job-template:
+    name: 'jbosstools-buildflow_{stream}'
+    disabled: '{obj:disabled}'
+    description: 'Build Flow that replaces the need for job cascading and interlinkage. Runs weekly. See <a href="https://issues.jboss.org/browse/JBIDE-13307">JBIDE-13307</a>'
+    jdk: '{default-jdk}'
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 10
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+    project-type: flow
+    dsl: |
+      parallel (
+      {{ ignore(FAILURE) {{ build("jbosstools-aerogear_{stream}") }} }},
+      {{ ignore(FAILURE) {{ build("jbosstools-base_{stream}") }} }}
+      )
```

Comment:
Playground is missing as a parallel task to base. Aerogear now runs after browsersim and browsersim-standalone.